### DISCUSSION TITLE
Disable plugins / middleware auto-loading

### DISCUSSION THREAD

**Discussion by vedmant** [nuxt/nuxt#16146] [Questions]
For example if I need to make a repo with multiple Nuxt apps that share the same root folder, plugins, middlewares, but still have some differences. With Nuxt 3 I have no choice but to use symbolic links or add empty js files with just `import` `export`. Is there any way to disable auto imports for things like plugins / middleware? Generally make it more configurable? Otherwise people will have to do any sort of tricks to do something more custom from basic Nuxt structure.

**Comment by TheAlexLichter** [comment]
Yes, AFAIK not possible right now. Feel free to raise an issue ☺️ 

As a workaround (for now), you can rename the plugin and middleware folders via `nuxt.config.ts` (`dir.middleware`, `dir.plugins`) to something that's not used and use the "typical" folders as before.

[Docs](https://nuxt.com/docs/api/configuration/nuxt-config#middleware)
